src/pages/guide-page/GuidePage.tsx

```diff
@@ -0,0 +1,43 @@
+import { Box, Card, CardContent, CardHeader, Container, makeStyles, Theme } from '@material-ui/core';
+import { CSSProperties } from '@material-ui/core/styles/withStyles';
+import { importMDX } from 'mdx.macro';
+import React from 'react';
+import { FCProps } from 'src/shared/types/FCProps';
+import { Styles } from 'src/shared/types/Styles';
+
+const Content = React.lazy(() => importMDX('./Content.mdx'));
```

Comment:
For multi-language supporting, I think we should create Content-fa.mdx, Content-en.mdx and with i18n.language import appropriate file.
something like this:
const Content = (lang => React.lazy(() => importMDX('./Content.mdx')))(i18n.language)();